content/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request.md

```diff
@@ -9,41 +9,41 @@ versions:
   github-ae: '*'
 ---
 
-Anyone with read permissions to a repository can create a pull request, but you must have write permissions to create a branch. If you want to create a new branch for your pull request and don't have write permissions to the repository, you can fork the repository first. For more information, see "[Creating a pull request from a fork](/articles/creating-a-pull-request-from-a-fork)" and "[About forks](/articles/about-forks)."
+Anyone with read permissions to a repository can create a pull request, but you must have write permissions to create a branch. If you want to create a new branch for your pull request and don't have write permissions to the repository, you can fork the repository first. The 'fork' function copies the repository to your profile, so that you can make changes to it on your own and submit them for review, without changing the master branch itself. For more information, read "[Creating a pull request from a fork](/articles/creating-a-pull-request-from-a-fork)" and "[About forks](/articles/about-forks)."
```

Comment:
I'd suggest using something other than `profile`, probably `account`.

Note that `master` should probably read `default`.
```suggestion
Anyone with read permissions to a repository can create a pull request, but you must have write permissions to create a branch. If you want to create a new branch for your pull request and don't have write permissions to the repository, you can fork the repository first. The 'fork' function copies the repository to your account, so that you can make changes to it on your own and submit them for review, without changing the default branch itself. For more information, read "[Creating a pull request from a fork](/articles/creating-a-pull-request-from-a-fork)" and "[About forks](/articles/about-forks)."
```